Makefile

```diff
@@ -5,13 +5,6 @@ package:
 	rm -Rf dist;\
 	python -m build
 
-pypi:
```

Comment:
There are no replacement for these commands right? (I guess they weren't commonly used?)

Reply:
https://github.com/kedro-org/kedro-plugins/blob/5ce4eabd721962e5f0a5514378318fe4508ba8f4/.github/workflows/check-release.yml#L91-L106

We don't manually upload to `pypi` anymore, this command was used as part of the CI months ago.